src/server/ApiRoutes/Hold.js

```diff
@@ -431,17 +407,18 @@ function createHoldRequestServer(req, res, pickedUpBibId = '', pickedUpItemId =
     docDeliveryData,
     itemSource,
     (response) => {
-      console.log('Hold Request Id:', response.data.data.id);
-      console.log('Job Id:', response.data.data.jobId);
+      const data = JSON.parse(response).data;
+      console.log('data', data);
```

Comment:
This might have been revised in another PR, but should these be logged as `DEBUG`?